src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -390,22 +390,30 @@ internal static bool IsPathInMemory(string filePath)
             // view of the current file or an untitled file.
             try
             {
-                // NotSupportedException or ArgumentException gets thrown when
-                // given an invalid path.  Since any non-standard path will
-                // trigger this, assume that it means it's an in-memory file
-                // unless the path starts with file:
-                Path.GetFullPath(filePath);
+                // File system absoulute paths will have a URI scheme of file:.
+                // Other schemes like "untitled:" and "gitlens-git:" will return false for IsFile.
+                var uri = new Uri(filePath);
+                isInMemory = !uri.IsFile;
```

Comment:
nit: maybe combine these into 1 line since `uri` isn't used? I don't feel strongly either way but it's something I thought about.

Reply:
Having `uri` as a variable makes it easier to inspect the entire URI when debugging - not just the resulting string.

Reply:
Good point 👍